libs/libc/signal/sig_emptyset.c

```diff
@@ -23,6 +23,7 @@
  ****************************************************************************/
 
 #include <signal.h>
+#include <nuttx/signal.h>
```

Comment:
ditto

Reply:
See comment with regard to sigaddset. I am assuming that the signal.h inclusion shourl remain.

Reply:
`nuttx/signal.h` includes `signal.h`